nais/dev-gcp.yaml

```diff
@@ -26,11 +26,14 @@ spec:
     scopes:
       consumes:
         - name: "altinn:serviceowner/reportees"
+  redis:
+    - instance: sessions
+      access: readwrite
+    - instance: lookup
```

Comment:
Har lest doc-en noen ganger nå, men tror det bare er et eksempel at de har to instances. Tipper vi bare trenger
```
- instance: lookup
  access: readwrite
```

Reply:
bra observert, det stemmer. copy paste feil fra meg. gikk for lookup som instance name

Reply:
kom ikke på noe bedre navn, men synes sessions var misvisende

Reply:
Siden den lever i namespace med andre apper: hva med altinn-rettigheter?

Reply:
uten dash da tenker jeg.. står ikke noe om hvordan det håndteres